experiments/AtmosLES/bomex_model.jl

```diff
@@ -13,7 +13,7 @@
 #
 # To simulate the full 6 hour experiment, change `timeend` to (3600*6) and type in
 #
-# julia --project experiments/AtmosLES/bomex.jl
+# julia --project experiments/AtmosLES/bomex_les.jl
```

Comment:
Maybe skip modifying this, since this should probably exist in the actual driver file--of which we now have several including LES, single stack, and edmf.